README.md

```diff
@@ -54,9 +54,10 @@ means = np.array([[0,0], [-5,5], [5,5]])
 X = np.vstack([np.random.randn(n_per_cluster, 2) + mu for mu in means])
 
 # Fit the data with BanditPAM:
-kmed = KMedoids(n_medoids = 3, algorithm = "BanditPAM")
+k = 3
+kmed = KMedoids(n_medoids = k, algorithm = "BanditPAM")
 # Writes results to gmm_log
-kmed.fit(X, 'L2', 3, "gmm_log")
+kmed.fit(X, 'L2', k, "gmm_log")
```

Comment:
The point of this issue is that passing the argument `k` here is redundant -- it is already passed in line 58. So, we can remove the passing of `k` in line 60.

Reply:
Hmmm, since the `fit` function requires four arguments, I may need to modify the python code then?

Reply:
We should neither require the parameter when instantiating the `KMedoids` object or when calling `.fit()`. Only if we call `.fit()` without the parameter, and the parameter has not already been set, should we raise an error. Could you modify the code to reflect this?